src/hooks/useDraftSpecEditor.ts

```diff
@@ -137,27 +139,44 @@ function useDraftSpecEditor(
         }
     }, [currentCatalog, draftSpecs, entityName, setSpecs]);
 
-    // This for advanced spec editor
+    // This for keeping advanced spec editor updated as the forms
```

Comment:
Pretty sure this is a combination of two ways to start this sentence: _This is to keep the advanced spec editor..._ **or** _For keeping the advanced spec editor..._